lib/crawlfs.js

```diff
@@ -9,22 +9,32 @@ async function determineFileType (filename) {
   const stat = await fs.lstat(filename)
   if (stat.isFile()) {
     return { type: 'file', stat }
-  } else if (stat.isDirectory()) {
-    return { type: 'directory', stat }
   } else if (stat.isSymbolicLink()) {
     return { type: 'link', stat }
+  } else if (stat.isDirectory()) {
+    return { type: 'directory', stat }
```

Comment:
any particular reason for swapping these?

Reply:
Uh, I think I was debugging something, can be swapped back